src/context/Global.tsx

```diff
@@ -470,6 +472,14 @@ const GlobalProvider = (props: { children: JSX.Element }) => {
         },
     );
 
+    const [showFiatAmount, setShowFiatAmount] = makePersisted(
+        // eslint-disable-next-line solid/reactivity
+        createSignal<boolean>(true),
```

Comment:
Let's disable that by default

Reply:
Why? I really think it's hyper subtle and noone will ever manually enable this as it's not discoverable.